controllers/btpoperator_controller_test.go

```diff
@@ -295,21 +268,27 @@ var _ = Describe("BTP Operator controller", Ordered, func() {
 				WithPolling(k8sOpsPollingInterval).
 				Should(Succeed())
 
-			time.Sleep(time.Second * 1)
```

Comment:
I tried to remove this ad-hoc sleep before webhook deletion and wanted to make it more deterministic but that surfaced another frequent flake, controller occasionally managed to sync webhooks back before `createResource()` got executed. Then I realized, instead of installing chart with webhooks just to delete webhooks after, it might be easier to have copy of the chart for tests without the webhooks in the first place - https://github.com/kyma-project/btp-manager/pull/94/commits/6acf3791903a470caf3a6eaebcd1c79b0e31251e

Reply:
@PK85 had better idea than commit the duplicate chart without webhooks in https://github.com/kyma-project/btp-manager/commit/6acf3791903a470caf3a6eaebcd1c79b0e31251e, there will instead be dynamic copy of it before the test execution.

Reply:
chart copy and filtering implemented in https://github.com/kyma-project/btp-manager/pull/94/commits/b84f6c7292c438adcf65f13fee729a990af3a113